manage.py

```diff
@@ -240,9 +240,49 @@ def toggle_site_status():
         print('Done!')
 
 
-# Please keep additional sets of commands down there
+@cli.group()
+@click.pass_context
+def highlevel(ctx):
+    """Stuff"""
```

Comment:
This docstring could be improved.

Reply:
whoops. that was me testing the output of `--help`